src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => (
+  <Link
+    to={`/people/${person.slug}`}
+    className={classNames({
+      'has-text-danger': person.sex === 'f',
```

Comment:
Create a enum for magic word and use them everywhere

Reply:
wdym?